### DISCUSSION TITLE
Formal support for Tauri (https://tauri.app/)?

### DISCUSSION THREAD

**Discussion by rkofman** [webdriverio/webdriverio#13672] [Ideas]
I love the first-class support for [Electron](https://www.electronjs.org/) as a [Desktop platform](https://webdriver.io/docs/desktop-testing/electron). I have recently been playing around with [Tauri](https://tauri.app/); which is like Electron but written in Rust, and uses a WebView (instead of packaging Chromium in every app). This results in much smaller distributable.

Tauri has a [pre-alpha version](https://github.com/tauri-apps/tauri/tree/dev/crates/tauri-driver) of an intermediary node that may be a good start for how the golden-path integration would work.

**Comment by christian-bromann** [comment]
Hey @rkofman 👋 

that's a great idea, would you like to help us on this effort? I am happy to create a repository within our community org for you to push code to. We have some funds we can give out for such contributions as well.

**Reply by rkofman** [reply]
I tried to shop it to existing maintainers of Tauri code/plugins on their discord, and nobody has taken the bait yet.

I'm new to Tauri, and if there is a financial incentive wanted to give others first dibs. However, I may move this forward when I find the time over the next few weeks. The current approach [documented by Tauri](https://v2.tauri.app/develop/tests/webdriver/example/webdriverio/) relies on a Rust crate that wraps Webdriver, perhaps unnecessarily. If I take this forward, I will try to duplicate a more `.js` centric approach as documented by a fellow user of Tauri [on their discord](https://discord.com/channels/616186924390023171/1290178898688213017/1290705626648805509).

tl;dr -- yes, please make me a repository. I will either hand it on to a responsible party, or build something there myself.

**Reply by christian-bromann** [reply]
Awesome 🎉 I just invited you to https://github.com/webdriverio-community/wdio-tauri-service. Happy to peer-program to get things started, just book a slot in https://webdriver.io/community/openofficehours
